cmd/lakectl/cmd/repo.go

```diff
@@ -129,66 +126,12 @@ var repoDeleteCmd = &cobra.Command{
 	},
 }
 
-var retentionCmd = &cobra.Command{
-	Use:    "retention [sub-command]",
-	Short:  "manage repository retention policies",
-	Hidden: true,
-}
-
-var getPolicyCmd = &cobra.Command{
```

Comment:
Do we want a short note somewhere, possibly on Slack so we don't keep a scary notice that only interests 1 or 0 people on documentation forever?